github/client.go

```diff
@@ -274,7 +274,12 @@ func (client *Client) FetchReleases(project *Project) (response []Release, err e
 		return
 	}
 
-	res, err := api.Get(fmt.Sprintf("repos/%s/%s/releases", project.Owner, project.Name))
+	request_url, err := url.Parse(fmt.Sprintf("repos/%s/%s/releases", project.Owner, project.Name))
```

Comment:
Two problems here:

1. We use camelCase for variable naming convention, so `request_url` should be `requestUrl`.

2. I dislike the fact that we construct a URL object out of this string, pass to `client.requestURL()` which takes a URL, and then extract the `.String()` value from the resulting URL. The whole String → URL → String dance now feels unnecessary. How about extracting the part of the logic from `client.requestURL()` into a new method that only prepends `/api/v3` to a string, and calling it something like `normalizeRequestPath()`?